src/wallet.js

```diff
@@ -223,6 +246,11 @@ const wallet = {
       xpubkey: privkey.xpubkey,
     });
 
+    if (multisig) {
+      const dxpriv = xpriv.deriveNonCompliantChild(`m/45'/${HATHOR_BIP44_CODE}'/0'`);
```

Comment:
This will not work if `xpriv` is already derived, right? Maybe we shouldn't handle this case for multisig, just throw Error. Better than to wrongly handle this.

Reply:
Done in [89d3a89](https://github.com/HathorNetwork/hathor-wallet-lib/pull/324/commits/89d3a89fb831ed94006445e3d29178170df31c29)